Client/src/Components/UserReview/DatasetModal.tsx

```diff
@@ -0,0 +1,85 @@
+import { Dialog, DialogActions, DialogContent, DialogTitle, Grid, Typography } from '@material-ui/core'
+import { FormikProps, FormikValues } from 'formik'
+import React, { useRef, useState } from 'react'
+
+import CancelIcon from "@material-ui/icons/Cancel"
+import { DatasetForm } from '../DatasetUpload/DatasetForm'
+import { DefaultFormFooter } from '../Forms/DefaultFormFooter'
+import { IApprovedDatasetModel } from '../../Models/Datasets/IApprovedDatasetModel'
+import { IDatasetModel } from '../../Models/Datasets/IDatasetModel'
+import { ModalFooter } from './ModalFooter'
+
+interface IProps {
+  open: boolean
+  dataset: IApprovedDatasetModel,
+  setOpen: (value: boolean) => void
+  handleApproveDataset: (datasetId: number) => void,
```

Comment:
![image](https://user-images.githubusercontent.com/22376976/108634410-763ad800-7447-11eb-8b00-c3ec210fc846.png)
 ModalFooter calls the callback and provides the dataset id as an argument, 

But the function provided ignores the argument
![image](https://user-images.githubusercontent.com/22376976/108634438-97032d80-7447-11eb-9ec8-5994c84fc8d2.png)
It looks like the argument can be safely removed

Reply:
Done